### PR TITLE
add support for VTE3 based terminals above v0.39

### PR DESCRIPTION
**Pull request by jck**

### CODE REVIEW

File: doc/togglecursor.txt [hunk 1]
```diff
@@ -18,13 +18,14 @@ supported terminals.
...
 
 Currently supported terminals are iTerm2 for the Mac (version 1.0.0.20130602
-beta or better is required) and KDE's Konsole.  The xterm console is partially
-supported as well.  Older xterm's didn't support the line cursor, so this plugin
-currently sets the cursor to underline instead.
-
-The gnome-terminal application doesn't support changing the cursor via escape
-sequences and is not supported.  On unsupported terminals, Vim's default
-behavior is left unaltered.
+beta or better is required), VTE3 based terminals(including gnome-terminal),
```
Comment (jszakmeister) [inline doc/togglecursor.txt]: You're missing a space between "terminals" and "(including".

File: plugin/togglecursor.vim [hunk 2]
```diff
@@ -38,8 +38,8 @@ let s:supported_terminal = ''
 if !has("gui_running")
     if $TERM_PROGRAM == "iTerm.app" || exists("$ITERM_SESSION_ID")
                 \ || $XTERM_VERSION != ""
-                " \ || $VTE_VERSION != ""
-        " iTerm, xterm, and future VTE based terminals support DESCCUSR.
+                \ || str2nr($VTE_VERSION) >= 3900
```
Comment (jszakmeister) [inline plugin/togglecursor.vim]: Nice to see support has finally shown up!  What are you running that you have access to this now?

**Comment by jszakmeister** [comment]
Would you mind fixing the commit message to capitalize the first letter and add a period to the end?  And it's not really "above v0.39".  It's more like "v0.39 or better", given your conditional, correct?  Thanks!

File: plugin/togglecursor.vim (hunk 2 above)
Comment (jck) [inline plugin/togglecursor.vim]: I use termite (https://github.com/thestinger/termite)

**Comment by jck** [comment]
Yes, it is v0.39 and above. I've fixed the commit message and added a space in the help text.
Feel free to amend my commit if you find any other issues :)

**Comment by jszakmeister** [comment]
Looks good!  Thank you.

**Pull request merged**